build.go

```diff
@@ -398,6 +398,8 @@ func (b *Builder) Build(s types.Storage, file string) error {
 
 				log.Infof("missing some cached layer output types, building anyway")
 			}
+		} else if len(binds) > 0 {
```

Comment:
Hrm, I had the cached && binds as I didn't want to emit the message if we didn't have a cache hit ... the first time building a layer that has binds we'll emit this message now, right?